friends/src/components/AddFriend/AddFriend.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from "react";
+import axios from "axios";
+
+const AddFriend = () => {
+  const [name, setName] = useState("");
+  const [age, setAge] = useState(null);
+  const [email, setEmail] = useState("");
+
+  const postFriend = () => {
+    const postObj = {
+      name: name,
+      age: age,
+      email: email
+    };
```

Comment:
When declaring an object, if the key and variable name are the same you can do as follows:

```
const postObj = {
  name,
  age,
  email
}
```